tests/test_append_to_inittab.rs

```diff
@@ -0,0 +1,32 @@
+#![cfg(feature = "macros")]
```

Comment:
Instead of `#[cfg]` on the test, I think you want:

```suggestion
#![cfg(all(feature = "macros", not(PyPy))]
```